sdk/openfeature-nodejs-provider/README.md

```diff
@@ -0,0 +1,109 @@
+# OpenFeature DevCycle NodeJS Provider
+
+This library provides a NodeJS implementation of the OpenFeature Provider interface for DevCycle.
+
+## Building
+
+Run `nx build sdk-openfeature-nodejs-provider` to build the library.
+
+## Running Unit Tests
+
+Run `nx test sdk-openfeature-nodejs-provider` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Example App
+
+See the [example app](/examples/openfeature-nodejs) for a working example of the OpenFeature DevCycle NodeJS Provider.
+
+## Usage
+
+See our [documentation](https://docs.devcycle.com/sdk/server-side-sdks/node/) for more information.
+
+```typescript
+import { OpenFeature, Client } from '@openfeature/js-sdk'
+import { DevCycleProvider } from '@devcycle/openfeature-nodejs-provider'
+import { initialize } from '@devcycle/nodejs-server-sdk'
+
+... 
+
+// Initialize the DevCycle SDK
+const dvcClient = await initialize(DVC_SERVER_SDK_KEY).onClientInitialized()
+// Set the initialzed DevCycle client as the provider for OpenFeature
+OpenFeature.setProvider(new DevCycleProvider(dvcClient))
+// Get the OpenFeature client
+openFeatureClient = OpenFeature.getClient()
+// Set the context for the OpenFeature client, you can use 'targetingKey' or 'user_id'
+openFeatureClient.setContext({ targetingKey: 'node_sdk_test' })
+
+
+// Retrieve a boolean flag from the OpenFeature client
+const boolFlag = await openFeatureClient.getBooleanValue('boolean-flag', false)
+```
+
+#### Passing DVCOptions to the DevCycleProvider
+
+Ensure that you pass any custom DVCOptions to the DevCycleProvider constructor
+
+```typescript
+const options = { logger: dvcDefaultLogger({ level: 'debug' }) }
+const dvcClient = await initialize(DVC_SERVER_SDK_KEY, options).onClientInitialized()
+OpenFeature.setProvider(new DevCycleProvider(dvcClient, options))
+```
+
+#### Required TargetingKey
+
+For DevCycle SDK to work we require either a `targetingKey` or `user_id` to be set on the OpenFeature context. 
+This is used to identify the user as the `user_id` for a `DVCUser` in DevCycle.
+
+#### Context properties to DVCUser
+
+The provider will automatically translate known `DVCUser` properties from the OpenFeature context to the `DVCUser` object.
```

Comment:
Could we link to a spec in the docs? Maybe this: https://docs.devcycle.com/sdk/client-side-sdks/javascript/javascript-gettingstarted#dvc-user-object